index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="asset/style/main.css" />
+    <!-- Google font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <title>Profsain Husseini</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a class="navMenu" href="#">Portfolio</a>
+        <a class="navMenu" href="#">About</a>
+        <a class="navMenu" href="#">Contact</a>
+      </nav>
+      <section id="homeSection">
+        <h1 class="headerTitle">
+          Hey there. I’m Profsain.
+        </h1>
+        <h2 class="headerTitle greenText">I’m a software developer</h2>
+        <p class="headerText">
+          I can help you build a product, feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="socialButton">
+          <div class="iconsBtn">
+            <img src="asset/image/icons/github.png" alt="github icon button" />
+          </div>
+          <div class="iconsBtn">
+            <img
+              src="asset/image/icons/linkedin.png"
+              alt="linkedin icon button"
+            />
+          </div>
+          <div class="iconsBtn">
+            <img src="asset/image/icons/Vector.png" alt="like icon button" />
+          </div>
+          <div class="iconsBtn">
+            <img
+              src="asset/image/icons/twitter.png"
+              alt="twitter icon button"
+            />
+          </div>
+          <div class="iconsBtn">
+            <img src="asset/image/icons/medium.png" alt="medium icon button" />
+          </div>
+        </div>
+      </section>
```

Comment:
- [ ] Kindly Remove the home section from the header tag since it's semantically inaccurate. You can either put it directly after the header tag or inside a `main` tag after the header tag the following ways:
```
<header>
...
</header>
<section>
...
</section>
```
or

```
<header>
...
</header>
<main>
  <section>
    ...
  </section>
</main>
```